packages/teams-js/src/public/storage.ts

```diff
@@ -0,0 +1,34 @@
+import { GlobalVars } from '../internal/globalVars';
+import { ensureInitialized } from '../internal/internalAPIs';
+import { HostClientType } from './constants';
+import { runtime } from './runtime';
+
+/**
+ * Contains functionality to allow web apps to store data in webview cache
+ *
+ * @beta
+ */
+export namespace storage {
```

Comment:
Upon further discussion of the feature, @MarcinJuraszek and I would prefer to call this `webStorage`. Can you also please confirm that when you refer to "web storage" in the comments and the function name, you are referring to this:  https://developer.mozilla.org/en-US/docs/Web/API/Web_Storage_API?

When renaming the capability, please also rename the file (webStorage.ts), test file (webStorage.spec.ts) and teams-test-app files (WebStorageAPIs.tsx).